extension/scripts/global/globalData.js

```diff
@@ -245,6 +245,7 @@ const DEFAULT_STORAGE = {
 		oldVersion: new DefaultSetting({ type: "string" }),
 		showNotice: new DefaultSetting({ type: "boolean", defaultValue: true }),
 	},
+	closedScopes: new DefaultSetting({ type: "array", defaultValue: [] }),

```

Comment:
Instead of adding yet another top level key, it's better to add it under `filters`.